metabase/metabase
# Convert `arrow.module.css` to CSS Modules

### Description

Converts `arrow.module.css` to CSS modules. There is only one place that this class is used, it might make more sense to put it into a local `.modules.css` file that's specific to the component. Let me know what you think.

### How to verify

Go to `Admin > Table Metadata > Segments` OR `Metrics`, then click `New Segment/Metric`. You should see under the `Data` element that there's a subtle arrow. That's the `arrowRight` class that has been converted to CSS modules. 

<img width="548" alt="image" src="https://github.com/metabase/metabase/assets/25306947/e9835724-4f59-43d8-9456-0948571ab66c">

### Checklist
